packages/combobox/test/lion-combobox.test.js

```diff
@@ -1575,6 +1575,211 @@ describe('lion-combobox', () => {
       });
     });
 
+    describe('Subclassers', () => {
```

Comment:
Nice :+1: 
Thanks a lot.

Could you maybe rename this describe to `'Override _getTextboxValueFromOption'`  and put it inside the Subclassers describe above?

If @jorenbroekema also agrees, it only needs:
- a reformat of this file, so that it passes the ci
- a changeset 
- a squash of the commits

Reply:
- `describe` is moved (be aware there are already several `describe('Subclassers'` in this file)
- changeset added
- squash done
- For the format, unfortunately I can't do it, I'm not able to find the proper way to make prettier work (see https://github.com/ing-bank/lion/issues/1388)

Reply:
Good catch about the multiple 'Subclassers' describes :) We will wrap up this mr and fix that later.

Prettier seems to work now (after a rebase with latest master, which pobably uses a later Prettier version).